docker-compose.yml

```diff
@@ -104,6 +104,7 @@ services:
       - DD_HOSTNAME=datadog
       - DD_DISABLED=true
       - DB_PATH=/app/.omg/data
+      - ETHEREUM_EVENTS_CHECK_INTERVAL_MS=800
```

Comment:
```suggestion
      - ETHEREUM_EVENTS_CHECK_INTERVAL_MS=8000
```

Reply:
why `8000`? it needs to be less then 1 second because thats the block production rate of the local geth